webapp-src/src/Login/Buttons.js

```diff
@@ -106,6 +106,15 @@ class Buttons extends Component {
       bContinue = <button type="button" className="btn btn-success" onClick={this.clickContinue} title={i18next.t("login.continue-title")}>
         <i className="fas fa-play btn-icon"></i>{i18next.t("login.continue")}
       </button>;
+      apiManager.glewlwydRequest("/auth/", "POST", {username: this.state.currentUser.username})
```

Comment:
As I said in this comment: https://github.com/yrammos/glewlwyd/commit/6dccebdb81856abd00774d3004e46fe2144b4871#r39940743 , you should not call `apiManager.glewlwydRequest` inside the `render()` function. What are you trying to achieve here?

Reply:
@babelouest thank you for the review.

The reproduction vector is described in the PR. To describe the symptom (not the cause) in brief: the green "Continue" button does not actually continue if the current user was made active automatically after another user logged out.